src/components/video-card/VideoCard.module.css

```diff
@@ -0,0 +1,63 @@
+.flex {
+  display: flex;
+}
+.videoCard {
+  display: flex;
+  flex-direction: column;
+  min-width: 5rem;
+  max-width: 20rem;
+  width: 100%;
+  gap: 4px;
+  padding: 8px;
+  box-shadow: rgba(0, 0, 0, 0.1) 0px 1px 3px 0px,
+    rgba(0, 0, 0, 0.06) 0px 1px 2px 0px;
+}
+.videoCard:hover {
+  opacity: 80%;
```

Comment:
change -> opacity: 0.8;

Reply:
Ah, yes. Will correct this. I don't know why my Chrome did not show this behaviour.